extensions/wikia/Search/classes/Test/Controller/SearchControllerTest.php

```diff
@@ -409,159 +409,32 @@ public function testHandleArticleMatchTrackingWithoutGoSearch() {
 		);
 	}
 
-
-
 	/**
-	 * @group Slow
-	 * @slowExecutionTime 0.08245 ms
 	 * @covers WikiaSearchController::pagination
 	 */
-	public function testPaginationWithoutConfig() {
+	public function testPaginationInvalidRequests() {
+		$extRes = $this->app->sendExternalRequest( WikiaSearchController::class, 'pagination' );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "pagination" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)